core/storage/user/gae_models.py

```diff
@@ -127,9 +127,7 @@ def get_lowest_supported_role():
 
     @staticmethod
     def get_deletion_policy():
-        """UserSettingsModel should be deleted after all the other models
-        belonging to the user are deleted or pseudonymized.
-        """
+        """Model contains data to delete corresponding to a user: id field."""
```

Comment:
For some of these models, other properties are also associated with users (such as usernames and emails). I suggest either being comprehensive in all of these docstrings, or indicating that because the id field is corresponding to users, other data in the model may also correspond to users (and thus the entire entity needs to be deleted).

Reply:
Yep, done.